### PR TITLE
Try out codeql

### PR DESCRIPTION
**Pull request by eriknordmark**

### CODE REVIEW

**Review by dautovri — APPROVED** [review]

**Comment by dautovri** [comment]
cpp experienced some issues.
What if we use to go and python only?

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
```
Comment (rene) [inline .github/workflows/codeql.yml:7]: Was this file auto-generated, right? I think we don't need this NOTE here....

**Comment by rene** [comment]
@eriknordmark would be nice to have a phony target like yetus to run codeql from command line with something like: `make codeql`

**Comment by eriknordmark** [comment]
This now runs - see https://github.com/lf-edge/eve/actions/runs/5531446462

File: .github/workflows/codeql.yml (hunk 1 above)
Comment (eriknordmark) [reply .github/workflows/codeql.yml]: Fixed

**Pull request merged**